From e3af2f4fecfe3fe1a054edc50c938a001810aa23 Mon Sep 17 00:00:00 2001
From: Punnaiah Choudary Kalluri <punnaiah.choudary.kalluri@xilinx.com>
Date: Mon, 18 Nov 2013 21:57:46 +0530
Subject: [PATCH 421/628] net: xilinx_emacps: Removed checkpatch errors

git://github.com/Xilinx/linux-xlnx.git xilinx-v2013.4
commit bfe268a32024096982e36a805d5e05b168b88d37

Removed checkpatch.pl errors.

Signed-off-by: Punnaiah Choudary Kalluri <punnaia@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
Signed-off-by: Liming Wang <liming.wang@windriver.com>
---
 drivers/net/ethernet/xilinx/xilinx_emacps.c |   11 ++++-------
 1 files changed, 4 insertions(+), 7 deletions(-)

diff --git a/drivers/net/ethernet/xilinx/xilinx_emacps.c b/drivers/net/ethernet/xilinx/xilinx_emacps.c
index 25a2640..25425ce 100644
--- a/drivers/net/ethernet/xilinx/xilinx_emacps.c
+++ b/drivers/net/ethernet/xilinx/xilinx_emacps.c
@@ -688,18 +688,15 @@ static void xemacps_adjust_link(struct net_device *ndev)
 				gmii2rgmii_reg |= XEMACPS_GMII2RGMII_SPEED1000;
 				xemacps_set_freq(lp->devclk, 125000000,
 						&lp->pdev->dev);
-			}
-			else if (phydev->speed == SPEED_100) {
+			} else if (phydev->speed == SPEED_100) {
 				regval |= XEMACPS_NWCFG_100_MASK;
 				gmii2rgmii_reg |= XEMACPS_GMII2RGMII_SPEED100;
 				xemacps_set_freq(lp->devclk, 25000000,
 						&lp->pdev->dev);
-			}
-			else if (phydev->speed == SPEED_10) {
+			} else if (phydev->speed == SPEED_10) {
 				xemacps_set_freq(lp->devclk, 2500000,
 						&lp->pdev->dev);
-			}
-			else {
+			} else {
 				dev_err(&lp->pdev->dev,
 					"%s: unknown PHY speed %d\n",
 					__func__, phydev->speed);
@@ -2030,7 +2027,7 @@ static int xemacps_start_xmit(struct sk_buff *skb, struct net_device *ndev)
 		return NETDEV_TX_BUSY;
 	}
 
-	if(xemacps_clear_csum(skb,ndev)) {
+	if (xemacps_clear_csum(skb, ndev)) {
 		spin_unlock_bh(&lp->tx_lock);
 		kfree(skb);
 		return NETDEV_TX_OK;
-- 
1.7.5.4

